tutorials/HDF5/Subset-HDF5-hyperspectral.Rmd

```diff
@@ -0,0 +1,241 @@
+---
+
+In this tutorial, we will subset an existing HDF5 file containing NEON
+hyperspectral data. The purpose of this exercise is to generate a smaller
+file for use in subsequent tuorials.
+
+<div id="ds-objectives" markdown="1">
+
+## Learning Objectives
+
+After completing this activity, you will be able to:
+
+* Navigate an HDF5 file to identify the variables of interest.</li>
+* Generate a new HDF5 file from a subset of the existing dataset.</li>
+* Save the new HDF5 file for future use.
+
+## Things You’ll Need To Complete This Tutorial
+To complete this tutorial you will need the most current version of R and, 
+preferably, RStudio loaded on your computer.
+
+### R Libraries to Install:
+
+* **rhdf5**: `install.packages("BiocManager")`, `BiocManager::install("rhdf5")`
+* **raster**: `install.packages('raster')`
+
+<a href="{{ site.baseurl }}/packages-in-r" target="_blank"> More on Packages in
+ R - Adapted from Software Carpentry.</a>
+
+### Data to Download
+The purpose of this tutorial is to reduce a large file (~`652Mb`) to a smaller size. The download linked here is the original large file, and therefore you may choose to skip this tutorial and download if you are on a slow internet connection or have file size limitations on your device.
+
+{% include/dataSubsets/_data_Imaging-Spec-Data-Full-Tile-H5-2020.html %}
+
+This file is NEON_D17_SJER_DP3_257000_4112000_reflectance.h5 from March 2019, available on data.neonscience.org.
+
+***
+{% include/_greyBox-wd-rscript.html %}
+***
+
+### Recommended Skills
+
+If you haven't already, we highly recommend you work through the 
+<a href="{{ site.baseurl }}/intro-hdf5-r-series" target="_blank"> *Introduction to Working with HDF5 Format in R* series</a>
+before moving on to this tutorial.
+
+</div>
+
+
+## Why subset your dataset? 
+There are many reasons why you may wish to subset you HDF5 file.
+Primarily, HDF5 files may contain a large amount of information
+that is not necessary for your purposes. By subsetting the file,
+you can reduce file size, thereby shrinking your storage needs,
+and shortening file transfer/download times. In this example, we
+will take a full HDF5 file of NEON hyperspectral reflectance data
+from the San Joaquin Experimental Range (SJER) that has a file size
+of ~`652 Mb` and make a new HDF5 file with a reduced spatial extent,
+and a reduced spectral resolution, yeilding a file of only ~`50.1 Mb`.
+This reduction in file size will make it easier and faster to share
+your data with others. We will then use this subsetted file in the
+<a href="{{ site.baseurl }}/intro-hsi-r-series" target="_blank"> *Introduction to Hyperspectral Remote Sensing Data* series</a>.
+
+If you find that downloading the full `652 Mb` file takes too much
+time or storage space, you will find a link to this subsetted file
+at the top of each lesson in the <a href="{{ site.baseurl }}/intro-hsi-r-series" target="_blank"> *Introduction to Hyperspectral Remote Sensing Data* series</a>.
+
```

Comment:
Two places below, I mention a section needed to help explore the data and set out a path for where the tutorial is headed so that the user understand why they are doing something and the goals.  I'm envisioning something like the following: 

## What to subset
(might move a bit of the above here).  

We're working with NEON data and we want a smaller file for data processing as we don't need all 426 bands and our study location is only in the NE corner of this tile.  Therefore, we're going to subset in two ways, by spectra and by spatial extent.  

### NEON HDF5 Format 
In order to subset, we need to figure out the elements of the file...

Reply:
Yeah! I think this will really improve this tutorial by explaining these steps. I will leave this 'conversation' open until these changed are made.

Reply:
Perfect!